test/sds/sds.mk

```diff
@@ -0,0 +1,102 @@
+
+
+# Target to run SDS graceful upgrade tests
+test-sds:
```

Comment:
are you planning to add to circleci?

Reply:
yes, but it's a long way to get there ( probably a week or so ) - some things are broken in agent.